include/wx/filehistory.h

```diff
@@ -60,15 +67,23 @@ class WXDLLIMPEXP_CORE wxFileHistoryBase : public wxObject
     void SetBaseId(wxWindowID baseId) { m_idBase = baseId; }
     wxWindowID GetBaseId() const { return m_idBase; }
 
+    void SetMenuLabelStyle(wxFileHistoryMenuLabelStyle style) { m_menuLabelStyle = style; }
+    wxFileHistoryMenuLabelStyle GetMenuLabelStyle() const { return m_menuLabelStyle; }
+
+    void RefreshLabels();
```

Comment:
Do we really need to make this one public, i.e. is there any point in calling it from elsewhere than from `SetMenuLabelStyle()`? I would rather refresh the labels automatically when the label style changes, why not? I.e. the usual rationale for not updating automatically is that you can have a series of `SetXXX()` calls and want to have only a single `Refresh()` at the end, but here this doesn't risk happening because there is just a single setter.